doc/rst/language/spec/task-parallelism-and-synchronization.rst

```diff
@@ -839,3 +843,11 @@ generates task according to normal Chapel rules.
 
    because the expression evaluated to determine whether to serialize
    always evaluates to true.
+
+.. _Yield_Task_Execution:
+
+Yielding Task Execution
+-----------------------
+
+Execution of the current task can be explicitly yielded with
+``currentTask.yield()``, providing an opportunity for other tasks to execute.
```

Comment:
A code example might be helpful here (particularly to clarify that no imports are required to use `currentTask`).

And this should say `currentTask.yieldExecution()`.

Reply:
Whoops, fixed to `currentTask.yieldExecution()`. For now I'd like to leave out code examples since I didn't want to figure out how into the weeds to get for scheduling policies and when/why this might be needed.